src/mlte/store/api/local/api.py

```diff
@@ -100,8 +100,10 @@ def _result_path(model_version_path: Path, result_identifier: str):
     :return: The formatted result path
     :rtype: Path
     """
-    path = (model_version_path / result_identifier).with_suffix(".json")
-    return Path(str(path).replace(" ", "-"))
+    path = (
+        model_version_path / result_identifier.replace(" ", "-")
+    ).with_suffix(".json")
+    return Path(str(path))
```

Comment:
The type of the above expression should also be `Path`, so there should be no need to cast to `str` and then immediately re-invoke the `Path` constructor. This can be:

```python
return (model_version_path / result_identifier.replace(" ", "-")).with_suffix(".json")
```

Reply:
Totally agree with that. Note that I didn't add that code, it was already that way. I was wondering why you were repacking that as a Path, and assumed you had a reason, so I prefered not to modify it. But since you don't seem to have a reason for that, I'll change it as you now suggest.

Reply:
And I answered my own question: it was that way because the replace didn't work on the Path, but on a string, hence needing the conversion, so it made sense. I've uploaded the updated version.